index.html

```diff
@@ -130,43 +146,56 @@ <h1 class="about_me_title">About Me</h1>
+      <ul class="list" id="languages"> 
+        <li class="main-li">Languages
+          <ul class="second-ul">
+            <li class="last-li">Javascript</li>
+            <li class="last-li">Ruby</li>
+            <li class="last-li">HTML</li>
+            <li class="last-li">CSS</li>
+          </ul>
+        </li>
       </ul>
-      <li class="frameworks"><img src="./images/Rectangle 56.svg">Frameworks</li>
-      <ul class="framewk">
-        <li class="li_framewk">React.js</li>
-        <li class="li_framewk">Ruby on Rails</li>
-        <li class="li_framewk">RSpec</li>
-        <li class="li_framewk">Capybara</li>
+      <ul class="list" id="frameworks">
+        <li class="main-li">Frameworks
+          <ul class="second-ul">
+            <li class="last-li">React.js</li>
+            <li class="last-li">Ruby on Rails</li>
+            <li class="last-li">RSpec</li>
+            <li class="last-li">Capybara</li>
+          </ul>
+        </li>
       </ul>
-      <li class="skills"><img src="./images/Rectangle 57.svg">Skills</li>
-      <ul class="sk">
-        <li class="li_skills">Database Management</li>
-        <li class="li_skills">Version Control</li>
-        <li class="li_skills">CLI</li>
-        <li class="li_skills">API Design</li>
-        <li class="li_skills">Web Development</li>
+      <ul class="list" id="skills">
+        <li class="main-li">Skills
+          <ul class="second-ul">
+            <li class="last-li">Database Management</li>
+            <li class="last-li">Version Control</li>
+            <li class="last-li">CLI</li>
+            <li class="last-li">API Design</li>
+            <li class="last-li">Web Development</li>
+          </ul>
+        </li>
       </ul>
-    </ul>
+    </div>
   </section>
   <section class="contact">
     <p class="contact_description">I'm always interested in hearing about new projects, so if you'd like to chat please
       get in touch.</p>
     <form class="contact_form" action="https://formspree.io/f/xqkoreon" method="POST">
-      <input type="text" class="name" placeholder="Full name" maxlength="30" required>
-      <input type="email" class="email" placeholder="Email address" required>
-      <textarea class="message" placeholder="Write me something..." maxlength="500" required></textarea>
-      <input type="submit" class="submit" value="Get in touch">
+      <fieldset class="name">
+        <input type="text" class="first-name" placeholder="Name" maxlength="30" required>
+        <input type="text" class="last-name" placeholder="Last Name" maxlength="50" required>
+      </fieldset>
+        <input type="email" class="email" placeholder="Email address" required>
+      <textarea class="message" placeholder="Write me something..." maxlength="500" row="10" column="50" required></textarea>
+      <input type="submit" class="resume" id="submit" value="Get in touch">
     </form>
   </section>
+  <hr class="footer-break">
   <footer class="footer">  
```

Comment:
- [ ] The social media icons of your footer and the button of the contact me is not displaying as can be seen in the figma desgin below. kindly adjust your design appropriately. It is important that your design matches correctly the provided figma template as this shows your attention to details as a frontend developer and this is a good skill to have :+1: 
- figma design
![fts](https://github.com/ommarz/Personal_Portfolio/assets/89055502/dadccb26-1de5-420a-9de8-e1bc7c80d58b)
- your design
![ftss](https://github.com/ommarz/Personal_Portfolio/assets/89055502/fe5efb72-e636-4368-9bb0-db8fdffe91dd)